docs/markdown/snippets/install_headers_preserve_path_arg.md

```diff
@@ -0,0 +1,35 @@
+## Added nobase arg to install_headers
+
+The `install_headers` function now has an optional argument `preserve_path`
```

Comment:
https://mesonbuild.com/Yaml-RefMan.html#linking-to-the-reference-manual

You can reference the function name as `[[install_headers]]` and it will do a cross-reference for you.